hw6/script.sh

```diff
@@ -0,0 +1,11 @@
+if ! mysql -h mariadb -u root -p believe -e "USE baseball"; then
```

Comment:
Nice check here to see if the baseball database exists. I do think it might be redundant having both if..else statement and CREATE DATABASE IF NOT EXISTS.  Your IF NOT EXISTS statement should be enough.